plane/source/docs/automatic-landing.rst

```diff
@@ -4,52 +4,79 @@
-During landing, the autopilot will shut down the throttle and hold the current heading when the plane reaches the flare point, controlled by the parameters described below.
+For many vehicles, most of the parameters associated with autolanding can be left at their defaults. The basic parameters are covered below in the Basic Autolanding Section for vehicles not using an airspeed sensor and/or rangefinder. For more advanced configuration, see the sections listed below:
 
-.. _automatic-landing_key_parameters:
+.. toctree::
+   :maxdepth: 1
 
-Key Parameters
---------------
+    Aborting an Autolanding <aborting-autolanding>
+    Precision Autolanding <precision-autolanding>
+    RangeFinder Autolanding <rangefinder-autolanding>
+    Using Reverse Thrust in Autolanding <reverse-thrust-autolanding>
+    Using DO_LAND_START <do-land-start>
+
+Basic Autolanding
+=================
+
+The phases of an autolanding are:
+
+- Passing final approach waypoint
+- Navigating to the flare point on a "glide-slope" (ie controlled descent)
+- Flare and touchdown
+
+.. image:: ../../../images/autolanding.jpg
+   :target: ../../_images/autolanding.jpg
+
+Setting Up the Approach Waypoint
+--------------------------------
+
+The autolanding begins after reaching the last navigation waypoint. This waypoint must be far enough from the touchdown point and at an altitude that a reasonable glide slope can be obtained. Without an airspeed sensor, the speed will be at least that of normal waypoint navigation and usually higher, due to the descent. So having a glide slope (angle between last waypoint and touchdown altitude) that is not only obtainable within the TEC_SINK_MAX rate of the vehicle, but well below that rate to avoid overspeeding which makes adjusting the flare more difficult, is required.
+
+In addition, it must be far enough away that any turns made by the mission to that waypoint, will have already be completed and the vehicle back on track in a line to the landing point. Usually, two waypoints are used. The first to allow any mission item (including aborted go-arounds for an autoland reattempt) to get aligned to another waypoint setup closer to the touchdown point but still providing a nice glide slope.
+
+.. image:: ../../../images/autolanding-approach.png
+    :target: ../../_images/autolanding-approach.png
+
+The above shows a planned approach with the the pre-approach waypoint 440m away from touchdown at 50m altitude, the final approach waypoint at 220m away and 25m altitude, giving a supposedly gentle glide slop of 6 degrees (which is a 1.7m/s descent at 15m/s airspeed). However, the actual result is 9 deg glide path since the start of the landing sequence was begun at the point shown, the the overshoots at the two waypoints results in a steeper glide slope (9 degrees). Spacing the waypoints further away would have allowed the approach to have settled by the final approach waypoint, with the desired glide slope.
```

Comment:
It's not clear which point is "the point shown" - there are 3 points shown on the diagram, all with the same white arrow. Perhaps use a different color for the problematic point?

Reply:
I will try to make it clearer